libraries/EEPROM/src/EEPROM.h

```diff
@@ -225,7 +225,11 @@ struct EEPROMClass {
   //Functionality to 'get' and 'put' objects to and from EEPROM.
   template< typename T > T &get(int idx, T &t)
   {
+#if defined (STM32L0xx)
+    EEPtr e = idx + DATA_EEPROM_BASE;
```

Comment:
EEPROM.h should not be modified, it is HIGH level API and generic.
Index 0 should not need to have an offset here, this should be managed in lower API.

Reply:
adding this offset in the eeprom_write_byte and eeprom_read_byte functions